persistence-modules/spring-data-jpa-query-2/src/test/java/com/baeldung/persistence/audit/EnversFooBarAuditIntegrationTest.java

```diff
@@ -123,24 +123,39 @@ public final void whenFooBarsModified_thenFooBarsAudited() {
         assertNotNull(barRevisionList);
         assertEquals(4, barRevisionList.size());
 
-        assertEquals("BAR", barRevisionList.get(0).getName());
-        assertEquals("BAR", barRevisionList.get(1).getName());
-        assertEquals("BAR1", barRevisionList.get(2).getName());
-        assertEquals("BAR1", barRevisionList.get(3).getName());
-
-        assertEquals(1, barRevisionList.get(0).getFooSet().size());
-        assertEquals(2, barRevisionList.get(1).getFooSet().size());
-        assertEquals(2, barRevisionList.get(2).getFooSet().size());
-        assertEquals(3, barRevisionList.get(3).getFooSet().size());
+        assertEquals("BAR", barRevisionList.get(0)
```

Comment:
I think we should keep the original formatting here.

Reply:
done.